wiki/History_of_osu!/2023/fr.md

```diff
@@ -0,0 +1,120 @@
+
+::: Infobox
+![](img/osu!monthly-9.jpg "Couverture de la neuvième édition d'osu!monthly")
+:::
+
+La troisième saison d'[osu!monthly](/wiki/Community/osu!monthly) s'est achevée avec la [neuvième edition](https://osu.ppy.sh/home/news/2023-06-26-osumonthly-9), retraçant les événements les plus importants de la communauté à partir de mars 2023. Principalement en raison de la dissolution de l'équipe de mapping au sein du projet [^mapping-team-disbanding] et de l'effort considérable nécessaire à la rédaction des éditions osu!monthly, l'équipe osu!monthly a décidé de mettre le projet en suspens pour le moment. La réorganisation du travail et la publication d'éditions saisonnières ou annuelles ont été envisagées lors de diverses discussions. [^monthly-discord][^writing-proposal] [0x84f](https://osu.ppy.sh/users/7944724), un des chefs de projet de la troisième saison, a aussi publié des sondages afin de savoir ce qui intéresait la communauté et ce qu'elle souhaitait voir dans la rubrique des actualités du site osu! en général.[^news-content-1][^news-content-2]
+
+En souvenir des deux premières saisons qui étaient publiées au format PDF, la neuvième édition du osu!monthly a été réalisée comme [un magazine](https://assets.ppy.sh/media/osu-monthly/2023-03/osu-monthly-2023-03.pdf) à la place des posts habituels. Il contenait des informations sur le mapping, skinning et les actualités de la communauté, ainsi que des interviews comme avec [EEEEEEEEEEEEEEE](https://osu.ppy.sh/users/2927048) et [pishifat](https://osu.ppy.sh/users/3178418).
+
+## Juillet
+
+Une [mise à jour du site](https://osu.ppy.sh/home/changelog/web/2023.725.0) a permis aux utilisateurs de mettre à jour le pays affiché sur leur profil, a condition qu'il ne coincide pas avec le pays duquel ils se sont connectés régulièrement au cours des six derniers mois, et que le joueur ne soit pas inscrit à un tournoi en cours. Auparavent, il était possible de modifier le pays affiché en contactant l'[équipe d'assistance aux comptes](/wiki/People/Account_support_team) et en fournissant une preuve de résidence, mais cette option a été abandonnée après décembre 2021 en raison de la transmission de données sensibles et de cas fréquents d'abus.[^flag-pr][^flag-issue][^flag-news]
+
+## Août
+
+De mai à août, le concours de skinning [Icons of an Era](https://osu.ppy.sh/home/news/2023-05-10-skinning-contest-icons-of-an-era) a été organisé pour célébrer les 15 ans d'osu! sous la forme de [skins](/wiki/Skin) représentants des beatmaps ou mascottes populaires. [Spoo](https://osu.ppy.sh/users/11805037), [Chromasia](https://osu.ppy.sh/users/7306251), et [thetasigma](https://osu.ppy.sh/users/6234482) ont respectivement remporté la première, deuxième et troisième place du concours. Le prix comprenait un badge personnalisé et des quantités variables d'osu!supporter.[^icons-of-an-era]
```

Comment:
représentants -> représentant

Reply:
In the english article it it written as "mascots" so i don't really know

Dans l'article anglais c'est marqué comme "mascots" donc en vrai je sais pas si je dois le laisser au pluriel

Reply:
ici "représentant" est au participe présent et au participe présent c'est invariable, alors pas de "s" à la fin du mot

Reply:
Ah oui pardon je sais juste pas lire j'ai cru que c'était mascotte le probleme ...